test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -54,15 +54,9 @@ class XdsOverrideHostTest : public LoadBalancingPolicyTest {
       : policy_(MakeLbPolicy("xds_override_host_experimental")) {}
 
   RefCountedPtr<LoadBalancingPolicy::Config> MakeXdsOverrideHostConfig(
-      bool exclude_draining_status = false,
+      Json::Array override_host_status = {"UNKNOWN", "HEALTHY", "DRAINING"},
```

Comment:
If you take my suggestion elsewhere about adding an `XdsHealthStatusSet` class, then we could use that type here as well.